kasa/tests/fixtures/HS100(UK)_1.0_1.2.6.json

```diff
@@ -0,0 +1,28 @@
+{
+    "system": {
+        "get_sysinfo": {
+            "active_mode": "schedule",
+            "alias": "Living Room Lamp",
+            "dev_name": "Wi-Fi Smart Plug",
+            "deviceId": "0000000000000000000000000000000000000000",
+            "err_code": 0,
+            "feature": "TIM",
+            "fwId": "00000000000000000000000000000000",
+            "hwId": "00000000000000000000000000000000",
+            "hw_ver": "1.0",
+            "icon_hash": "",
+            "latitude": 0,
+            "led_off": 0,
+            "longitude": 0,
+            "mac": "00:00:00:00:00:00",
+            "model": "HS100(UK)",
+            "oemId": "00000000000000000000000000000000",
+            "on_time": 4102,
+            "relay_state": 1,
+            "rssi": -58,
+            "sw_ver": "1.2.6 Build 200727 Rel.120236",
+            "type": "IOT.SMARTPLUGSWITCH",
+            "updating": 0
+        }
+    }
```

Comment:
Should we also have discovery info for this?

Reply:
This doesn't have discovery info as it's legacy.  I was planning to use the lack of a `discovery_result` in the fixture as an indicator of legacy.  Adding `discovery_result` to fixtures for legacy seemed unnecessary as it's more or less the same as get_sysinfo which we already have in all the fixtures.  Plus we'd have to update all the existing fixtures somehow.

Reply:
Ah, true, makes sense.. Please remove adding the new file from this PR and create a separate one for it, then this is good to go :+1: